File: tests/fixtures/Sample.java
package com.example.sample;

import java.util.List;
import java.util.Optional;
import java.util.Map;

/**
 * Maximum retry count.
 */
public class Sample {

    public static final int MAX_RETRIES = 3;

    private int counter = 0;

    /**
     * A user in the system.
     */
    public static class User {
        public String id;
        public String name;
        public String email;

        public User(String id, String name, String email) {
            this.id = id;
            this.name = name;
            this.email = email;
        }
    }

    /**
     * Status of an entity.
     */
    public enum Status {
        ACTIVE,
        INACTIVE
    }

    /**
     * A service interface.
     */
    public interface Service {
        void init();
        Optional<User> findUser(String id);
    }

    /**
     * Greet a person by name.
     */
    public static String greet(String name) {
        return "Hello, " + name + "!";
    }

    /**
     * Fetch data from a URL.
     */
    public byte[] fetchData(String url) throws Exception {
        return new byte[0];
    }

    private void privateHelper() {
        // internal
    }
}

File: tests/fixtures/Sample.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;

namespace Example.Sample
{
    /// <summary>
    /// Maximum retry count.
    /// </summary>
    public static class Constants
    {
        public const int MaxRetries = 3;
    }

    /// <summary>
    /// A user in the system.
    /// </summary>
    public class User
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }

        public User(string id, string name, string email)
        {
            Id = id;
            Name = name;
            Email = email;
        }
    }

    /// <summary>
    /// Status of an entity.
    /// </summary>
    public enum Status
    {
        Active,
        Inactive
    }

    /// <summary>
    /// A service interface.
    /// </summary>
    public interface IService
    {
        void Init();
        User FindUser(string id);
    }

    /// <summary>
    /// Service for managing users.
    /// </summary>
    public class UserService : IService
    {
        private readonly object _db;

        public UserService(object db)
        {
            _db = db;
        }

        public void Init()
        {
            // initialization
        }

        /// <summary>
        /// Find a user by ID.
        /// </summary>
        public User FindUser(string id)
        {
            return null;
        }

        /// <summary>
        /// Greet a person.
        /// </summary>
        public static string Greet(string name)
        {
            return $"Hello, {name}!";
        }

        /// <summary>
        /// Fetch data asynchronously.
        /// </summary>
        public async Task<byte[]> FetchData(string url)
        {
            return Array.Empty<byte>();
        }

        private void PrivateHelper()
        {
            // internal
        }
    }
}
